src/components/ArtistName/ArtistName.css

```diff
@@ -0,0 +1,14 @@
+.artist-name h3 {
+    font-family: Helvetica;
+    color: #a0a0a0;
+    font-size: 11px;
```

Comment:
Such a detail thing to be commenting on (I know) but according to instructions the artist name should be in 14px!